src/functions/template-tags/general.php

```diff
@@ -725,3 +725,30 @@ function tribe_set_time_limit( $limit = 0 ) {
 		return @set_time_limit( $limit );
 	}
 }
+
+if ( ! function_exists( 'tribe_context' ) ) {
+	/**
+	 * A wrapper function to get the singleton, immutable, global context object.
+	 *
+	 * Due to its immutable nature any method that would modify the context will return
+	 * a clone of the context, not the original one.
+	 *
+	 * @since TBD
+	 *
+	 * @return Tribe__Context The singleton, immutable, global object instance.
+	 */
+	function tribe_context() {
+		$context = tribe( 'context' );
+
+		/**
+		 * Filters the global context object.
+		 *
+		 * @since TBD
+		 *
+		 * @param Tribe__Context $context The singleton, immutable, global object instance.
+		 */
+		$context = apply_filters( 'tribe_context', $context );
```

Comment:
Should we call this  `tribe_global_context`? 
Otherwise it make confuse people into thinking that would allowing filtering values of the context.